src/zsf_config.c

```diff
@@ -73,7 +84,19 @@ void zsf_config_unload(zsf_config_t *config_ptr) {
       config_ptr->num_locks--;
       unload_csv(&config_ptr->locks[config_ptr->num_locks].timeseries_data);
       free(config_ptr->locks[config_ptr->num_locks].operational_parameters_file);
+      free(config_ptr->locks[config_ptr->num_locks].id);
     }
   }
   return;
 }
+
+// Find sealock in config by id.
+// Returns -1 if not found.
+sealock_index_t zsf_config_get_lock_index(zsf_config_t *config_ptr, char *lock_id) {
```

Comment:
Can both parameters be pointers to const?

Reply:
Fixed